### PR TITLE
Added (44) Top DSA Interview Question.

### LINKED ISSUE #614: [Feature Request] Top DSA Interview Questions Topic
### Description:
I propose adding a curated list of top Data Structures and Algorithms (DSA) interview questions to  platform. This feature will help users prepare for technical interviews, enhancing their coding skills and increasing their chances of success.

**Would you like to work on this feature?**  
- [ yes ] Yes  
- [ no ] No

### PR DESCRIPTION
**Pull request by Meetpidev**
### Description
Added 44 Top DSA Interview Question.

### Solved Issue
Fixes #614

### Type of change
New feature (non-breaking change which adds functionality)

### Checklist:
[ yes ] My code follows the style guidelines of this project
[ yes ] I have performed a self-review of my code
[ yes ] I have commented my code, particularly in hard-to-understand areas
[ yes ] I have added tests that prove my fix is effective or that my feature works
[ yes ] New and existing unit tests pass locally with my changes
[ yes ] Any dependent changes have been merged and published in downstream modules

### proof of changes
image
image
image

### CODE REVIEW

**Comment by Meetpidev** [comment]
@ajay-dhangar I update PR.. please Review it..

File: src/pages/dsa-interview/index.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import { motion } from "framer-motion";
+import Layout from "@theme/Layout";
+import problemsData from "../../data/problemData";
+import '../../css/index.css';
```
Comment (ajay-dhangar) [inline src/pages/dsa-interview/index.tsx:5]: delete line number 5 ( `import '../../css/index.css';`)

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
Please delete the line `import '../../css/index.css';` because the file `../../css/index.css` is not in our site.

**Comment by Meetpidev** [comment]
@ajay-dhangar i added index.css.. its for styling dsa-interview.. please review it...

File: src/pages/dsa-interview/index.tsx (hunk 1 above)
Comment (Meetpidev) [reply src/pages/dsa-interview/index.tsx:5]: i added index.css

**Pull request closed without merging**